test/unit/text/shaping.cpp

```diff
@@ -65,39 +65,39 @@ TEST_CASE("shaping")
     mapnik::face_manager fm(fl, font_file_mapping, font_memory_cache);
 
     {
-        std::vector<std::tuple<unsigned, unsigned>> expected =
-            {{{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}}};
+        std::vector<std::pair<unsigned, unsigned>> expected =
+            {{0, 0}, {0, 3}, {0, 4}, {0, 7}, {3, 8}, {11, 9}, {68, 10}, {69, 11}, {70, 12}, {12, 13}};
         // with default NotoSans-Regular.ttc and NotoNaskhArabic-Regular.ttf ^^^
```

Comment:
nope, didn't work across all compilers - reverted back to good old pair 791f7b51bbf99bbd303367965b53364709f58af7